### PR TITLE
[RUNTIME][CRT] init TVMPackedFunc's name

### PR DESCRIPTION
**Pull request by windclarion**
or else src/runtime/crt/graph_runtime/graph_runtime.c TVMGraphRuntime_Run
Line 639 will show messy character.

### CODE REVIEW

**Comment by tqchen** [comment]
cc @liangfu @areusch

File: src/runtime/crt/common/packed_func.c [hunk 1]
```diff
@@ -85,6 +85,7 @@ int TVMPackedFunc_InitGlobalFunc(TVMPackedFunc* pf, const char* name, const TVMA
     return status;
   }
 
+  strncpy(pf->name, name, sizeof(pf->name));
```
[the remark this answers is not in the document]
Comment (windclarion) [reply src/runtime/crt/common/packed_func.c]: done!

File: src/runtime/crt/common/packed_func.c [hunk 2]
```diff
@@ -85,6 +86,7 @@ int TVMPackedFunc_InitGlobalFunc(TVMPackedFunc* pf, const char* name, const TVMA
     return status;
   }
 
+  snprintf(pf->name,  sizeof(pf->name), "%s", name);
```
Comment (liangfu) [inline src/runtime/crt/common/packed_func.c:89]: looks like the extra space has to be removed to pass clang-format.

**Comment by windclarion** [comment]
just rebase to retrigger, I can't understand fail message

**Comment by tqchen** [comment]
This is a known flaky error, please retrigger again

**Comment by liangfu** [comment]
Thanks @windclarion for contributing. Thanks @tqchen for reviewing. This is now merged.

**Pull request merged**